plugin/trino-druid/src/main/java/io/trino/plugin/druid/DruidJdbcClient.java

```diff
@@ -135,6 +135,11 @@ public ResultSet getTables(Connection connection, Optional<String> schemaName, O
     @Override
     public Optional<ColumnMapping> toColumnMapping(ConnectorSession session, Connection connection, JdbcTypeHandle typeHandle)
     {
+        Optional<ColumnMapping> mapping = getForcedMappingToVarchar(typeHandle);
+        if (mapping.isPresent()) {
+            return mapping;
+        }
+
```

Comment:
You might also want something like below at the end of the type mappings to ensure unmapped types can be returned as varchars. The current code you've added only maps types from a given list to varchars.

See PostgreSqlClient for example.

```java
if (getUnsupportedTypeHandling(session) == CONVERT_TO_VARCHAR) {
    return mapToUnboundedVarchar(typeHandle);
}
```